pkg/sql/codegen/pai/tensorflow.go

```diff
@@ -112,6 +112,39 @@ func TFLoadAndExplain(ir *ir.ExplainStmt, session *pb.Session, modelPath string,
 	return code.String(), nil
 }
 
+// TFLoadAndEvaluate generates PAI-TF evaluate program.
+func TFLoadAndEvaluate(ir *ir.EvaluateStmt, session *pb.Session, modelPath string, expn *ExplainRender) (string, error) {
+	var tpl = template.Must(template.New("Evaluate").Parse(tfEvaluateTmplText))
+	ossModelDir := OSSModelURL(modelPath)
+	paiExplainTable := ""
+	if tensorflow.IsPAI() && ir.TmpEvaluateTable != "" {
+		paiExplainTable = ir.TmpEvaluateTable
+	}
+
+	validationMetrics := ""
+	for k, v := range ir.Attributes {
+		// NOTE: only validation.metrics attribute is supported for TO EVALUATE clause.
+		if k == "validation.metrics" {
+			validationMetrics = v.(string)
+		}
+	}
+
+	filler := evaluateFiller{
+		OSSModelDir:       ossModelDir,
+		DataSource:        session.DbConnStr,
+		Select:            ir.Select,
+		ResultTable:       ir.Into,
+		IsPAI:             tensorflow.IsPAI(),
+		PAITable:          paiExplainTable,
+		ValidationMetrics: validationMetrics,
```

Comment:
There're so many `Filler`s like this, later we have to find a better way to organize these.